.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: main
+on:
+  - push
+  - pull_request
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: stable
```

Comment:
Seems like `go.mod` here is [referencing go1.13](https://github.com/VictoriaMetrics/metricsql/blob/72b650d3218bc79ec559f5a4567338ab18fa6e7d/go.mod#L3), should the workflow use the same version or it's fine to keep with the latest stable?

Reply:
I'm not rly sure why we don't bump go.mod version in this package.
But my thinking is that it is even good that we test this package with `stable` version of go. Since it is used in other projects that are likely running stable go.